### PR TITLE
changes made to menu alignment

### PR DESCRIPTION
**Pull request by iiCe89**
## Checklist

- [x] I ran `npm test` locally and it passed without errors.

- [ ] I only edited the `_data/pixels.json` file.

- [ ] I entered the `username` in the `pixels.json` that I'm also using to create this pull request.
- [x] I acknowledge that all my contributions will be made under the project's [license](../../LICENSE.md).

## Description

I aligned menu items to center and made the hover red to jazz it up , this has been tested locally on all devices and there is no menu issue breaking. Updated file **index.njk** & **main.css**

## Related issues

List any related issues here

### CODE REVIEW

File: index.njk [hunk 1]
```diff
@@ -30,35 +30,25 @@
...
-          </li>
-          <li class="nav-item">
-            <a class="nes-btn external-link" href="https://github.com/twilio-labs/open-pixel-art" target="_blank" rel="noreferrer" aria-label="Check on Github">View on GitHub</a>
-          </li>
-        </ul>
-      </nav>
     </header>
-
+    <section class="nes-container is-centered">
+    <div class="row">
```
Comment (dkundel) [inline index.njk]: This one is actually less accessible. Could you please use the `nav` element and an unordered list? You should be able to style the `nav` element the way you want to

File: styles/main.css [hunk 2]
```diff
@@ -148,3 +148,12 @@ p code {
   font-size: 10px;
   border: 1px solid black;
 }
+
+.menu-btn:hover{
+  background-color:#f22f46;
```
Comment (dkundel) [inline styles/main.css]: can we also change the inset here?

```
.menu-btn:hover::after {
box-shadow: inset -4px -4px #730000;
}
```

Comment (dkundel) [inline styles/main.css]: Can we change the `color` on hover to white?

File: index.njk (hunk 1 above)
Comment (iiCe89) [reply index.njk]: Yup sure will get it resolved as soon as i can

File: styles/main.css (hunk 2 above)
Comment (iiCe89) [reply styles/main.css]: yup sure thing

Comment (iiCe89) [reply styles/main.css]: sure

**Pull request closed without merging**